### PR TITLE
Rework required for rebase on .org django bootstrap Docker image

### PR DESCRIPTION
**Pull request by cobusc**
Achtung! Currently the .org Django bootstrap image changes the ownership of the source directory and all files and directories underneath it when running in docker compose. This is definitely not ideal. I need to figure out if it is the celery instance or the auth service container doing this (for gunicorn and nginx), or whether it is both.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,22 +1,25 @@
-FROM praekeltfoundation/python-base:3.6
+FROM praekeltfoundation/django-bootstrap:py3
 
 ARG EXTRA_DEPS
 
 ENV DJANGO_SETTINGS_MODULE=project.settings
+ENV SKIP_MIGRATIONS=1
```
Comment (AltusBarry) [inline Dockerfile]: Is 0 then to run them? Just checking for QA setup.

File: Dockerfile [hunk 2]
```diff
@@ -1,22 +1,25 @@
...
 
 # Git is required because one of the pip requirements is pulled from github.
 RUN apt-get update && apt-get install -y git gcc netcat $EXTRA_DEPS gettext libgettextpo-dev
 
-RUN mkdir /static
+RUN mkdir -p /app/static
 
-WORKDIR /app/
+COPY . /app/
```
Comment (AltusBarry) [inline Dockerfile]: Don't copy the app before we install requirements, if possible. Requirement layer will be cached unless the requirements change. But the actual app is likely to change each deploy and will bust all cached layers beneath it.

File: Dockerfile (hunk 1 above)
Comment (cobusc) [reply Dockerfile]: No. You need to omit the environment variable entirely. I'll remove this from here and add to the `docker-compose.yml` file in `core-general`.

File: Dockerfile (hunk 2 above)
Comment (cobusc) [reply Dockerfile]: The requirements are not installed under the `/app` directory, so it should not matter. It is installed in the global python scope and `/usr/local/src`.

**Pull request merged**